public/js/question.js

```diff
@@ -36,13 +36,21 @@ var sendAnswerRequest = function(ans) {
         url: '/submitanswer',
         data: { questionId: questionId, answer: ans },
         success: function(data) {
-            swal({
-                title: 'Correct',
-                text: 'Congratulations! You gained ' + data.points + ' points!',
-                type: 'success'
-            }, function () {
-                window.location.href = '/';
+
+            $('#modalAlert').modal({
+                dismissible: false,
+                opacity: 0.5,
+                complete: function() {
+                    if(getRating() > 0) {
+                        submitQuestionRating(getRating(), questionId);
+                    }
+                    window.location.href = '/';
+                }
             });
+
+            $('#modalAlertMsg').html('Congratulations! You gained ' + data.points + ' points!<br>Please rate the difficulty of this question:');
```

Comment:
`<br>` -> `<br/>`

Reply:
HTML5 standard is <br>

<br/> was XHTML standard which came after HTML4 but before HTML5.